spec/models/form_spec.rb

```diff
@@ -354,9 +354,9 @@ def answer_local_authority(lettings_log)
       expect(form.sections[0].class).to eq(Form::Sales::Sections::Setup)
       expect(form.subsections.count).to eq(1)
       expect(form.subsections.first.id).to eq("setup")
-      expect(form.pages.count).to eq(18)
+      expect(form.pages.count).to eq(12)
```

Comment:
I'd support just setting these to test for positive values rather than specific counts like we have in `form_handler_spec`

```suggestion
      expect(form.pages.count).to be_positive
```

Reply:
Imma keep as is, cause the setup section doesn't change much (and it shouldn't), so maybe thips test failing would make people think twice before adding/removing things from setup